src/licensedcode/data/rules/gpl-2.0-plus_with_bison-exception-2.2_3.RULE

```diff
@@ -0,0 +1,34 @@
+  Distributed under the OSI-approved BSD License (the "License");
```

Comment:
As a convention we name the rule after the license expression...
So may be use something like `bsd-new_and_gpl-2.0-plus_with_bison-exception_....`
 (within reason, unless there many many licenses)

Reply:
Done.